Gemfile

```diff
@@ -31,6 +31,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'secure_headers', '~> 3.0.0'
 gem 'sidekiq'
 gem 'simple_form', github: 'amoose/simple_form', branch: 'feature/aria-invalid'
+gem 'sinatra', require: false
```

Comment:
I added that so we can view the [Sidekiq admin UI](https://github.com/18F/identity-idp/pull/30/files#diff-21497849d8f00507c9c8dcaf6288b136R3) on the remote servers. We can remove that when we're closer to a production ready app or add it at a later point/separate PR.